File: app/src/main/java/ua/fvadevand/screenlock/MainActivity.java
package ua.fvadevand.screenlock;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.support.v4.content.LocalBroadcastManager;

import ua.fvadevand.screenlock.services.LockScreenAccessibilityService;

public class MainActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        if (getIntent().getAction() != null && getIntent().getAction().equals(LockScreenAccessibilityService.ACTION_SCREEN_LOCK)) {
            LocalBroadcastManager.getInstance(this).sendBroadcast(new Intent(LockScreenAccessibilityService.ACTION_SCREEN_LOCK));
        }
        finish();
    }
}

File: app/src/main/java/ua/fvadevand/screenlock/ui/MainActivity.kt
package ua.fvadevand.screenlock.ui

import android.app.Activity
import android.content.Intent
import android.os.Bundle

import androidx.localbroadcastmanager.content.LocalBroadcastManager

import ua.fvadevand.screenlock.services.LockScreenAccessibilityService

class MainActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        if (intent?.action == LockScreenAccessibilityService.ACTION_SCREEN_LOCK) {
            LocalBroadcastManager.getInstance(this)
                .sendBroadcast(
                    Intent(LockScreenAccessibilityService.ACTION_SCREEN_LOCK)
                        .setPackage(packageName)
                )
        }
        finish()
    }
}
